### DISCUSSION TITLE
Hard Fail/ Soft Fail

### DISCUSSION THREAD

**Discussion by AlPhaSh3ll** [postalserver/postal#1529] [Help with using Postal]
Hi am trying to send 5k emails from my postal that is hosted on Contabo ubuntu 20.
I am not able to send emails to gmail mx from postal am getting hard bounce can you guys please help?

**Comment by adamcooke** [comment]
What error are you getting back from Gmail? You should see it in the web UI in Postal.

**Reply by AlPhaSh3ll** [reply]
August 04, 2021 16:24
HARD FAIL
Permanent SMTP delivery error when sending to outlook-com.olc.protection.outlook.com (104.47.22.161) Recipient added to suppression list (too many hard fails).

550 5.7.1 Unfortunately, messages from [ip] weren't sent. Please contact your Internet service provider since part of their network is on our block list (S3140). You can also refer your provider to http://mail.live.com/mail/troubleshooting.aspx#errors. [AM7EUR06FT037.eop-eur06.prod.protection.outlook.com]

Time: 0.12s

Support Ref: W49SVZHQ




August 05, 2021 12:42
HARD FAIL
Permanent SMTP delivery error when sending to aspmx.l.google.com (2607:f8b0:4001:c16::1b)

550-5.7.1 [Ip] Our system has detected that this message is 550-5.7.1 likely suspicious due to the very low reputation of the sending 550-5.7.1 domain. To best protect our users from spam, the message has been 550-5.7.1 blocked. Please visit 550 5.7.1 https://support.google.com/mail/answer/188131 for more information. v9si1462759ilu.149 - gsmtp

Time: 0.39s

Support Ref: G1EINWED

**Reply by AlPhaSh3ll** [reply]
i am getting this error hard fail when i send email to outlook  and Gmail.

**Reply by adamcooke** [reply]
The errors are all there in those messages.

* network is on our block list
* likely suspicious due to the very low reputation of the sending domain

What sort of messages are you sending?

**Reply by AlPhaSh3ll** [reply]
its a normal message for email marketing nothing else if you want me to show you i can write here

**Reply by AlPhaSh3ll** [reply]
and also my emails are sending very slow from postal

**Reply by AlPhaSh3ll** [reply]
> The errors are all there in those messages.
> 
> * network is on our block list
> * likely suspicious due to the very low reputation of the sending domain
> 
> What sort of messages are you sending?

i am waiting for your response

**Reply by adamcooke** [reply]
I really have nothing else to add. Your IP and domain are just poor quality according to Google and Microsoft. This isn't anything to do with Postal. If you can't debug that, you'll need to find a email provider who can handle that for you.
